raster-test/src/test/scala/geotrellis/raster/io/geotiff/LocalBytesStreamSpec.scala

```diff
@@ -0,0 +1,81 @@
+import geotrellis.raster._
+import geotrellis.raster.testkit._
+import geotrellis.raster.io.geotiff._
+
+import java.nio.ByteBuffer
+import java.nio.file.{Paths, Files}
+
+import spire.syntax.cfor._
+import org.scalatest._
+
+class LocalBytesStreamerSpec extends FunSpec
+  with Matchers
+  with GeoTiffTestUtils {
+
+  describe("Streaming bytes locally") {
+    val path = geoTiffPath("ls8_int32.tif")
+    val geoTiffBytes = Files.readAllBytes(Paths.get(path))
+    val buffer = ByteBuffer.wrap(geoTiffBytes)
+    val chunkSize = 2000
+    val local = new MockLocalArrayBytes(chunkSize, geoTiffBytes) 
```

Comment:
I guess that didn't really occur to me when I was writing it... I got rid of it, though.